modules/init.d.js

```diff
@@ -0,0 +1,16 @@
+function init_d() {

```

Comment:
That's an error from moving the file, and not using git rm. It was working on my local repository because only one of the files were there.
The one in root needs to be deleted.